src/components/autocomplete/Autocomplete.vue

```diff
@@ -8,6 +11,7 @@
             :loading="loading"
             :rounded="rounded"
             :icon="icon"
+            :icon-right="clearButton ? 'close-circle' : ''"
```

Comment:
Maybe this could be a computed function, WDUT?

Reply:
Yeah, just changed it and it looks a lot cleaner. Thanks